### DISCUSSION TITLE
Microprofile specification proposal update

### DISCUSSION THREAD

**Discussion by amoscatelli** [t1/problem-details#68] [General]
Hi @t1 !
Are you planning to update the microprofile specification proposal ?
https://github.com/eclipse/microprofile-sandbox/tree/master/proposals/problem-details

Again, I am very interested in this spec.
How is this going to move further ?

**Comment by t1** [comment]
Hi @amoscatelli !

I've been using mainly GraphQL in my projects for the last three years. And as the interest has been... minimal, I think I won't find the time required to keep leading this project. Maybe you want to take over? I do agree that this is important to the Java-REST community.

**Reply by amoscatelli** [reply]
I dunno for how long I can do this, because I may switch to GraphQL sooner or later.
Anyway I can co-lead for the required time to update the spec and so some necessary mantainance.

**Reply by amoscatelli** [reply]
@t1 what do you think ?

**Reply by t1** [reply]
I guess this is just how things work 🤪 please go ahead as far as you can! ❤️

**Reply by amoscatelli** [reply]
Should I fork from : 
https://github.com/t1/microprofile-sandbox
https://github.com/t1/problem-details
?

Or should I fork from master and manually report changes ?
https://github.com/eclipse/microprofile-sandbox

**Reply by t1** [reply]
The last one... and write a mail to the MP mailing list. Then you'll be able to work without having to wait for me. But that's only the spec and tck. You'll also need an implementation, and for that you can fork t1/problem-details.

**Reply by amoscatelli** [reply]
I sent a subscription request for : microprofile-dev
